plugins/generator-1.19.4/forge-1.19.4/procedures/item_get_cooldown.java.ftl

```diff
@@ -1,2 +1,2 @@
 <#include "mcitems.ftl">
-(${input$entity} instanceof Player _plrCldRem${cbi} ? (double) _plrCldRem${cbi}.getCooldowns().getCooldownPercent(${mappedMCItemToItem(input$item)}, 0f) : 0d)
\ No newline at end of file
+(${input$entity} instanceof Player _plrCldRem${cbi} ? _plrCldRem${cbi}.getCooldowns().getCooldownPercent(${mappedMCItemToItem(input$item)}, 0f) * 100d : 0d)
```

Comment:
getCooldownPercent returns float so don't increase precision for no reason, but instead multiply cleanly by * 100 and use type marker.

Also no need for 0d as 0 is the same